### ISSUE TITLE
My version of "Arduino ST4"  for my NEQ5

### ISSUE THREAD

**Issue by DominiqueMACHU** [kevinferrare/arduino-st4#7]
Hello Kevin,
A big thank you for your project, I realized this winter my version based on an ARDUINO UNO R3 (ch340) for my mount NEQ5.
I just finished the tests everything is OK.
I come to publish the realization of this project on GitHut :

https://github.com/DominiqueMACHU/Interface-Arduino-ST4-for-NEQ5

I look forward to your new version.

friendships

Dominique (Lyon-France)

**Comment by kevinferrare** [comment]
Hi Dominique,

I would be interested in seing what you did, but the repository your link points to only has a readme.md file.

Did you commit your files?

Kind regards,
Kevin

**Comment by DominiqueMACHU** [comment]
Hi Kevin,

The file is in the branch "hardware", i copy them in master branch.
Excuse me it's my first utilisation of GITHUB!

Kind regards,

Dominique

**Comment by kevinferrare** [comment]
Oh!

I see you made a nice PDF manual explaining how to build the hardware and how to use it in various softwares.
However I see no code although I see that the configuration window is different from your screenshots.
Did you change the code?

**Comment by DominiqueMACHU** [comment]
No,  I did not change the code, it's version 1 of your original code.

In my first version I had cabled the Rj12 as the standard ST4 :
"ST-4 style guide ports (sometimes referred to as home ports or CCD ports) look
like the diagram shown to the right. The pins are connected as:
Pin 1: Not usually connected
Pin 2: Common (ground)
Pin 3: RA +
Pin 4: Dec +
Pin 5: Dec-
Pin 6: RA
"
-->  and all orders were reversed!
I had modified your code by inverting the controls and the telescope go well on the targets.
Finally I rewired the case "Dual Axis" and install your original code V1.
The telescope goes well on its targets in GOTO mode, the only problem encountered in your version is the passage of the meridian where the DEC commands are reversed!
I am waiting for your official version for this modification, in your ASCOM driver? I think ?.

Thank you for your project this interface ST4 is a + to find the objects of the sky since a star of reference.

Sincerely
